templates/02-executable-org.md

```diff
@@ -85,5 +87,6 @@ The number (#) should represent the number of proposals the Org has submitted to
 
 >❗️If applicable, please include a report and brief retrospective on spending throughout the previous year.
 
-## Fund Management
+## Management of Org Funds
```

Comment:
Should we just say "Management of Funds" here? Instead of just Org Funds?

Reply:
Yeah this was meant to Management of Funds...and I swear I updated the PR to reflect that but maybe I didn't save it correctly.